.github/workflows/CI.yml

```
@@ -59,17 +57,14 @@ jobs:
         with:
           version: '1'
           show-versioninfo: true
-      - run: |
-          julia --project=docs -e '
-            using Pkg
-            Pkg.develop(PackageSpec(path=pwd()))
-            Pkg.instantiate()'
-      - run: |
-          julia --project=docs -e '
-            using Documenter: doctest
-            using Distributions
-            doctest(Distributions)'
-      - run: julia --project=docs docs/make.jl
+      - name: Install the current package into the `docs` environment
+        run: 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
```

Shouldn't it be possible to simplify this to

```suggestion
      - name: Instantiate the `docs` environment
        run: 'using Pkg; Pkg.instantiate()'
```

due to the `sources` setting in docs/Project.toml?

---

Oops! You're right. I had forgotten about that :sweat_smile: